upgrading/topics/keycloak/changes.adoc

```diff
@@ -1,5 +1,18 @@
 == Migration Changes
 
+=== Migrating to 11.0.0
+
+==== LDAP no-import bugfix
+
+In the previous {project_name} version, when LDAP provider was configured with `Import Users` OFF, then it was possible to update the
+user even if some of non-LDAP mapped attributes were changed. This resulted in confusing behaviour, when the attribute appeared to be updated,
```

Comment:
Also follow "this" noun.   "This situation"
US spelling for words ending in "our" is "or" --- "behavior"

Reply:
Changed both things as you suggested.